Source/WebKit/UIProcess/Launcher/glib/ProcessLauncherGLib.cpp

```diff
@@ -211,6 +211,20 @@ void ProcessLauncher::launchProcess()
     GRefPtr<GSubprocessLauncher> launcher = adoptGRef(g_subprocess_launcher_new(G_SUBPROCESS_FLAGS_INHERIT_FDS));
     g_subprocess_launcher_take_fd(launcher.get(), socketPair.client, socketPair.client);
 
+#if USE(SYSPROF_CAPTURE)
+    int sysprofFd = -1;
+
+    if (const char* sysprofFdStr = getenv("SYSPROF_CONTROL_FD"))
+        sysprofFd = atoi(sysprofFdStr);
+
+    if (sysprofFd > 0) {
```

Comment:
While it's extremely unlikely to happen here, file descriptor zero is a valid one, so the check here should use `>= 0`:

```suggestion
    if (sysprofFd >= 0) {
```